charter/README.md

```diff
@@ -102,6 +102,16 @@ Chairs have responsibilities to their parent Working Group: they ensure that the
 
 Chairs have responsibility to all participants in transparently reporting the group's activity.
 
+Chairs also have the following regular duties within their Working Groups:
+* Collect agenda items prior to each meeting to ensure a smooth and focused Working Group meeting.
+* Ensure that the Working Group is notified of the agenda for the meeting in a timely manner advance of the meeting, and that if there is no agenda, the meeting is altered or cancelled accordingly.
+* Attend and lead each Working Group meeting.
+    * If the Chair is unable to attend a given meeting, they should delegate a pre-appointed person to lead the meeting in their stead.
+* Ensure meeting notes are uploaded to the `governance` repo in the Working Group's respective sub-folder following each meeting.
+* Check into persistent issues present on a given Working Group's repositories of responsibility (ex. bot failures)
+
+Each Working Group may also establish more specific Chair responsibilities within their domains.
```

Comment:
I would prefer to keep a single _chair_. The one person ultimately responsible, but that person may delegate most, if not all, of their duties to other members of the group. The intent is to avoid the bystander-effect. Two chairs can never be sure who is ultimately responsible for what. I have no objection to the idea, just the name. Using _sub-chair_, _chief_, _lead_, or anything else would be fine.